pageserver/src/tenant/timeline/walreceiver/connection_manager.rs

```diff
@@ -609,6 +609,17 @@ impl ConnectionManagerState {
     fn register_timeline_update(&mut self, timeline_update: SafekeeperTimelineInfo) {
         WALRECEIVER_BROKER_UPDATES.inc();
 
+        info!(
```

Comment:
Let's also make this debug/trace to not log each second.

Reply:
Done